events/templates/home.html

```diff
@@ -1,11 +1,15 @@
 {% extends 'base.html'%}
 {% load crispy_forms_tags %}
   {%block content%}
+
     <h1 class="">Events</h1>
     <form class="d-inline" action="{% url 'home'%}" method="get">
       <!-- {% csrf_token %} -->
       <fieldset>
-        {{ form|crispy }}
+        {{ form.search|as_crispy_field }}
+        {{ form.show_old|as_crispy_field }}
+        {% include "widgets/sortOptions.html" %}
```

Comment:
Does the choice field not render properly?
If this is about search persistence, I'll worry about that later this week. It requires generating a filled form when searching, then passing the form to the render. I think you did something like that in the view file.

Reply:
Ah saw your comment about styling. I haven't run it yet. I'll give that a shot when I can.

Reply:
Just to comment to give you a head's up before you try it out. If I do go through cirspy forms(without any additional styling), the sort by options can't be in a line. I tried adding attributes but it seems crispy overrides whatever classes I add. And IMO i think the button toggle styling doesn't look too shabby.

Reply:
This will do for now.
The 'django' way of doing things is to define a widget in the widgets.py file that calls to your template. This would require overriding 2 classes though ([StackOverflow](https://stackoverflow.com/questions/11556143/customizing-django-forms-with-radioselect-widget/15951741)).
The RadioSelect field loops over its choices and calls to an option class to render each option. We would want to modify the option class. It's a bit involved. I can't figure it out.

Reply:
Wait I see what you mean. Crispy overrides the template. You gotta go even farther and define an __init__ method to  the form in order to call the custom widget after the initial render ([more here](https://simpleisbetterthancomplex.com/tutorial/2018/11/28/advanced-form-rendering-with-django-crispy-forms.html)).
I like the buttons, but maybe a select field (dropdown) is easier than calling the form fields individually. Then again, we only have to work on this code base for another 2 weeks so it doesn't really matter. form > function